identity/client/src/pages/groups/modals/EditModal.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import { FC, useEffect, useState } from "react";
+import { EditFormModal, UseEntityModalProps } from "src/components/modal";
+import useTranslate from "src/utility/localization";
+import { useApiCall } from "src/utility/api/hooks";
+import { useNotifications } from "src/components/notifications";
+import { Group, updateGroup } from "src/utility/api/groups";
+import TextField from "src/components/form/TextField";
+
+const EditModal: FC<UseEntityModalProps<Group>> = ({
+  entity: group,
+  open,
+  onClose,
+  onSuccess,
+}) => {
+  const { t } = useTranslate();
+  const { enqueueNotification } = useNotifications();
+
+  const [callUpdateGroup, { namedErrors, loading }, reset] =
+    useApiCall(updateGroup);
+
+  const [name, setName] = useState("");
```

Comment:
The default state has to be the current name of the group instead of empty.

```suggestion
  const [name, setName] = useState(group.name);
```

In this modal it works by using `useEffect` [here](https://github.com/camunda/camunda/pull/19368/files#diff-a80ea604c479005fa382e193f258d1a2c800f46f83ec4d258a720edff3381c0fR48), which is actually not needed any more since the entity modals not only accept non-null objects as the entity (so the group will always be available when the modal is mounted)